test/functional/lua/buffer_updates_spec.lua

```diff
@@ -365,129 +367,129 @@ describe('lua: nvim_buf_attach on_bytes', function()
-        check_events {
-          {'test1', 'bytes', 1, 3, 0, 15, 15, 1, 0, 1, 0, 1, 1};
-        }
-
-        feed '3J'
-        check_events {
-          {'test1', 'bytes', 1, 5, 0, 31, 31, 1, 0, 1, 0, 1, 1};
-          {'test1', 'bytes', 1, 5, 0, 47, 47, 1, 0, 1, 0, 1, 1};
-        }
+    it("single and multiple join", function()
+      local check_events = setup_eventcheck(verify, origlines)
+      feed "ggJ"
+      check_events {
+        { "test1", "bytes", 1, 3, 0, 15, 15, 1, 0, 1, 0, 1, 1 },
+      }
+
+      feed "3J"
+      check_events {
+        { "test1", "bytes", 1, 5, 0, 31, 31, 1, 0, 1, 0, 1, 1 },
+        { "test1", "bytes", 1, 5, 0, 47, 47, 1, 0, 1, 0, 1, 1 },
+      }
     end)
 
-    it('opening lines', function()
-        local check_events = setup_eventcheck(verify, origlines)
-        -- meths.buf_set_option(0, 'autoindent', true)
-        feed 'Go'
-        check_events {
-          { "test1", "bytes", 1, 4, 7, 0, 114, 0, 0, 0, 1, 0, 1 };
-        }
-        feed '<cr>'
-        check_events {
-          { "test1", "bytes", 1, 5, 7, 0, 114, 0, 0, 0, 1, 0, 1 };
-        }
+    it("opening lines", function()
+      local check_events = setup_eventcheck(verify, origlines)
+      -- meths.buf_set_option(0, 'autoindent', true)
+      feed "Go"
+      check_events {
+        { "test1", "bytes", 1, 4, 7, 0, 114, 0, 0, 0, 1, 0, 1 },
+      }
+      feed "<cr>"
+      check_events {
+        { "test1", "bytes", 1, 5, 7, 0, 114, 0, 0, 0, 1, 0, 1 },
+      }
     end)
 
-    it('opening lines with autoindent', function()
-        local check_events = setup_eventcheck(verify, origlines)
-        meths.buf_set_option(0, 'autoindent', true)
-        feed 'Go'
-        check_events {
-          { "test1", "bytes", 1, 4, 7, 0, 114, 0, 0, 0, 1, 0, 5 };
-        }
-        feed '<cr>'
-        check_events {
-          { "test1", "bytes", 1, 4, 7, 0, 114, 0, 4, 4, 0, 0, 0 };
-          { "test1", "bytes", 1, 5, 7, 0, 114, 0, 0, 0, 1, 4, 5 };
-        }
+    it("opening lines with autoindent", function()
+      local check_events = setup_eventcheck(verify, origlines)
+      meths.buf_set_option(0, "autoindent", true)
+      feed "Go"
+      check_events {
+        { "test1", "bytes", 1, 4, 7, 0, 114, 0, 0, 0, 1, 0, 5 },
+      }
+      feed "<cr>"
+      check_events {
+        { "test1", "bytes", 1, 4, 7, 0, 114, 0, 4, 4, 0, 0, 0 },
+        { "test1", "bytes", 1, 5, 7, 0, 114, 0, 0, 0, 1, 4, 5 },
+      }
     end)
 
-    it('setline(num, line)', function()
+    it("setline(num, line)", function()
       local check_events = setup_eventcheck(verify, origlines)
       funcs.setline(2, "babla")
       check_events {
-        { "test1", "bytes", 1, 3, 1, 0, 16, 0, 15, 15, 0, 5, 5 };
+        { "test1", "bytes", 1, 3, 1, 0, 16, 0, 15, 15, 0, 5, 5 },
       }
 
-      funcs.setline(2, {"foo", "bar"})
+      funcs.setline(2, { "foo", "bar" })
       check_events {
-        { "test1", "bytes", 1, 4, 1, 0, 16, 0, 5, 5, 0, 3, 3 };
```

Comment:
Can this conversion be disabled? I like the semantic distinction between `,` (between fields of a tuple) and `;` (after item in a ordered list of tuples) in places like this.

Reply:
No, stylua is an "opinionated" formatter like python's black, so this isn't an option. We could fork or vendor stylua if we wanted to do this